src/storage/query/QueryBaseProcessor-inl.h

```diff
@@ -260,8 +260,10 @@ nebula::cpp2::ErrorCode QueryBaseProcessor<REQ, RESP>::getSpaceEdgeSchema() {
 }
 
 template <typename REQ, typename RESP>
-nebula::cpp2::ErrorCode QueryBaseProcessor<REQ, RESP>::checkExp(
-    const Expression* exp, bool returned, bool filtered, bool updated, bool allowNoexistentProp) {
```

Comment:
Update will report error when property is not existed.

Reply:
Em, indeed, good catch... Has graphd verified it or not? If not, maybe we can't delete it simply

Reply:
What if I handle separately in UpdateProcessor, WDYT?

Reply:
As I known, it won't.

Reply:
What's the mistake of origin implementation?

Reply:
Ok, I will check it and keep the checkExp as simple as possible

Reply:
Before https://github.com/vesoft-inc/nebula/pull/4270, all field must exists in expression. This has make storage hard to handle property not existent... #4565 and https://github.com/vesoft-inc/nebula-ent/issues/1165